music21/stream/base.py

```diff
@@ -6468,22 +6471,15 @@ def makeNotation(self,
                     tiePitchSet=tiePitchSet,
                     inPlace=True,
                     **srkCopy)
-        # environLocal.printDebug(['makeNotation(): meterStream:', meterStream, meterStream[0]])
-        try:
-            measureStream.makeTies(meterStream, inPlace=True)
-        except StreamException as e:
-            strE = str(e)
-            environLocal.warn('StreamException: makeTies: ' + strE)
+
+        measureStream.makeTies(meterStream, inPlace=True)
 
         # measureStream.makeBeams(inPlace=True)
         if not measureStream.streamStatus.beams:
             try:
                 measureStream.makeBeams(inPlace=True)
-            except (StreamException, meter.MeterException):
-                # this is a result of makeMeasures not getting everything
-                # note to measure allocation right
-                # environLocal.printDebug(['skipping makeBeams exception', StreamException])
-                pass
+            except meter.MeterException as me:
+                environLocal.warn(['skipping makeBeams exception', me])
```

Comment:
I like making this a warning.

I do think that I want to get away from environLocal.X() with a list attribute.  Never quite understood it (wasn't my dev.)